### PR TITLE
Added schedule back

### PR DESCRIPTION
**Pull request by kjustin2**

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -2791,7 +2791,7 @@
 			font-size: 0.9em;
 			font-weight: 400;
 			padding: 0 0.75em 0.75em 0.75em;
-			text-align: left;
+			text-align: center;
```
Comment (RitwikGupta) [inline assets/css/main.css:2794]: Any reason for this design change?

File: index.html [hunk 2]
```diff
@@ -75,12 +75,16 @@ <h2>This is our challenge to you.</h2>
...
                                     <div class="content">
+										<!--
                                         <header class="major">
                                             <h2>Schedule: TBD</h2>
                                         </header>
+										-->
                                         <h3><b>Location:</b> <strong>The William Pitt Union, 3959 Fifth Ave., Pittsburgh, PA 15213</strong></h3>
                                         <p></p>
-										<!--
+										<header class="major">
```
Comment (RitwikGupta) [inline index.html]: The indentation here is way off

**Review by RitwikGupta — CHANGES_REQUESTED** [review]

File: assets/css/main.css (hunk 1 above)
Comment (kjustin2) [reply assets/css/main.css:2794]: The table did not fit on the page right for some reason with left text-align.

File: index.html (hunk 2 above)
Comment (kjustin2) [reply index.html]: I am completely unsure as to why whenever I upload to Github my indentations keep getting thrown wayyyyyyy off. I can fix this up.

File: index.html [hunk 3]
```diff
@@ -281,11 +281,11 @@ <h3><b>Medical Education</b></h3>
                                         <h3><b>Big-Data in Healthcare</b></h3>
                                         <p>Microsoft Azure Gift Card</p>
                                     </li>
-									                  <li>
+								  <li>
```
Comment (RitwikGupta) [inline index.html:284]: I'm going to merge this, but please fix your editors to use soft tabs with 4 spaces. Use a formatter like Emmett to ensure proper formatting.

**Review by RitwikGupta — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (kjustin2) [reply index.html:284]: My editor already does use 4 space soft tabs, but something weird keeps happening when I upload to Github that throws the spacing way off. I'm looking at how to fix it.

**Pull request merged**